packages/components/src/components/Button/tests/Button.test.tsx

```diff
@@ -0,0 +1,73 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconIcon } from '@frontify/fondue-icons';
+import { render } from '@testing-library/react';
+import { describe, expect, it, vi } from 'vitest';
+
+import { Button, type ButtonStyle } from '..';
+
+const BUTTON_TEXT = 'Frontify';
+const BUTTON_TEST_ID = 'button';
+
+const styles: ButtonStyle[] = ['default', 'positive', 'negative', 'danger', 'loud'];
+
+describe('Button component', () => {
+    for (const style of styles) {
```

Comment:
Not a good practices to add a for each like this as it doesn't really test anything.

Also shouldn't the test be in Playwright?

Reply:
fixed